### PR TITLE
Fix Markdown to adapters documentation

### PR DESCRIPTION
**Pull request by bacarini**
From:
selection_009

To:
selection_010

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
it looks like there is an extra blank row at the top. Is there a way to get rid of that?

**Comment by bacarini** [comment]
There is not @NullVoxPopuli ... It is the header. I tried to fix without it but I couldn't do it.
Any idea on what to add on this header?

**Comment by beauby** [comment]
Add a normal row, followed by a row of dashes (`-----`) in order to fill the [header](https://help.github.com/articles/github-flavored-markdown/#tables).

File: docs/general/adapters.md [hunk 1]
```diff
@@ -91,11 +95,13 @@ ActiveModel::Serializer::Adapter.register(:special_adapter, MyAdapter)
 
 ### Looking up an adapter
 
-| `ActiveModel::Serializer::Adapter.adapter_map` | A Hash of all known adapters { adapter_name => adapter_class } |
-| `ActiveModel::Serializer::Adapter.adapters`    | A (sorted) Array of all known adapter_names |
-| `ActiveModel::Serializer::Adapter.lookup(name_or_klass)` |  The adapter_class, else raises an `ActiveModel::Serializer::Adapter::UnknownAdapter` error |
-| `ActiveModel::Serializer::Adapter.adapter_class(adapter)` | delegates to `ActiveModel::Serializer::Adapter.lookup(adapter)` |
-| `ActiveModel::Serializer.adapter` | a convenience method for `ActiveModel::Serializer::Adapter.lookup(config.adapter)` |
+| Method |Return value|
```
Comment (beauby) [inline docs/general/adapters.md]: Could you add spaces around "Return value"?

File: docs/general/adapters.md [hunk 2]
```diff
@@ -91,11 +95,13 @@ ActiveModel::Serializer::Adapter.register(:special_adapter, MyAdapter)
...
+| :------------ |:---------------|
+|`ActiveModel::Serializer::Adapter.adapter_map` | A Hash of all known adapters `{ adapter_name => adapter_class }` |
```
Comment (beauby) [inline docs/general/adapters.md]: Could you add a space between "|" and "`ActiveModel..."?

**Comment by beauby** [comment]
Also, if you could align the middle and right "|", that would be perfect :+1:

File: docs/general/adapters.md (hunk 2 above)
Comment (bacarini) [inline docs/general/adapters.md]: :+1:

**Pull request merged**